pulpcore/pulpcore/app/settings.py

```diff
@@ -197,21 +197,21 @@
             },
         }
     },
-    'server': {
-        'working_directory': '/var/lib/pulp/tmp',
+    'SERVER': {
+        'WORKING_DIRECTORY': '/var/lib/pulp/tmp',
     },
-    'content': {
-        'web_server': 'django',
-        'host': None,
+    'CONTENT': {
+        'WEB_SERVER': 'django',
+        'HOST': None,
```

Comment:
I think this usage needs to be changed

https://github.com/pulp/pulp/blob/3.0-dev/pulpcore/pulpcore/app/serializers/fields.py#L175